src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/AesImplementation.OpenSsl.cs

```diff
@@ -62,12 +62,20 @@ private int KeyWrap(ReadOnlySpan<byte> source, Span<byte> destination, int enc)
 
                     IntPtr algorithm = GetKeyWrapAlgorithm(keySizeInBits);
 
-                    return Interop.Crypto.EvpCipherCreate(
+                    SafeEvpCipherCtxHandle ctx = Interop.Crypto.EvpCipherCreate(
                         algorithm,
                         ref MemoryMarshal.GetReference(key),
                         key.Length * 8,
                         ref MemoryMarshal.GetReference(ReadOnlySpan<byte>.Empty),
                         enc);
+
+                    if (ctx.IsInvalid)
```

Comment:
I did originally have a call to Interop.Crypto.CheckValidHandle (or whatever it is), but I clearly lost it during refactoring. I would say "good catch", but since you got here via debugging crashes, I'll just say "whoops!"